presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -155,6 +157,10 @@
     private boolean adaptiveFilterReorderingEnabled = true;
     private boolean zstdJniDecompressionEnabled;
 
+    private Duration fileStatusCacheExpireAfterWrite = new Duration(1, TimeUnit.MINUTES);
+    private long fileStatusCacheMaxSize = 1000 * 1000;
+    private List<String> fileStatusCacheTables = ImmutableList.of();
```

Comment:
This being a config seems to be a bit inflexible for making changes if the hot table changes as workload shifts, since it would require a cluster restart. Yet I don't see an easy way besides using a mysql table for dynamic configuration (like `DbResourceGroupConfigurationManager`), so maybe we should just keep it as is.

Reply:
yep, seems the only way, unless having external dependency